lib/migration.pm

```diff
@@ -143,6 +144,13 @@ sub disable_installation_repos {
     }
 }
 
+# Based on bsc#1097111, need to disable kernel multiversion before migration, and enable it after migration
+# https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-update-preparation.html#sec-update-preparation-multiversion
+sub disable_kernel_multiversion {
+    my $sed_para = check_var('VERSION', get_var('ORIGIN_SYSTEM_VERSION')) ? 's/^multiversion/#multiversion/g' : 's/^#multiversion/multiversion/g';
```

Comment:
I do not prefer using this tricky to do enable multiversion within disable_kernel_multiversion, we need keep simple and stupid for function.